CAM/ShakeCinematicCam.md

```diff
@@ -5,14 +5,22 @@ ns: CAM
 
 ```c
 // 0xDCE214D9ED58F3CF 0x61815F31
-void SHAKE_CINEMATIC_CAM(char* p0, float p1);
+void SHAKE_CINEMATIC_CAM(char* shakeType, float amplitude);
 ```
 
-```
-p0 argument found in the b617d scripts: "DRUNK_SHAKE"  
-```
+Makes the idle and/or cinematic camera shake.
+
+Shake Types:
+SMALL_EXPLOSION_SHAKE
+MEDIUM_EXPLOSION_SHAKE
+LARGE_EXPLOSION_SHAKE
+HAND_SHAKE
+JOLT_SHAKE
+VIBRATE_SHAKE
+WOBBLY_SHAKE
+DRUNK_SHAKE
```

Comment:
These are being copied almost everywhere, wouldn't it be better to make a list somewhere and centralize by referring to that list? If the list ever needs to be maintained it's going to be easier to keep track of a single file 😊

Reply:
hey, i have no idea where to make that list, what do you suggest?